sim/core/runic_power_helper.go

```diff
@@ -134,7 +134,7 @@ func (rp *RunicPowerBar) CorrectBloodTapConversion(sim *Simulation, bloodGainMet
 	// so in english
 	// 1. try to convert active blood rune -> death rune
 	// 2. if no active blood, convert inactive blood rune -> death rune
-	// 3. then convert one inactive death rune -> active
+	// 3. then convert one inactive blood or death rune -> active
 
 	slot := int8(-1)
 	if rp.runeStates&isSpentDeath[0] == 0 {
```

Comment:
I want to confirm this logic, I’m pretty sure what happens in game is it takes the first blood and converts it, not sure it prioritized active runes

Reply:
confirmed in game and fixed code. Blood Tap just goes from left to right and checks for the first rune off cooldown, it doesn't prioritize active runes